app/styles/_core.less

```diff
@@ -281,7 +281,7 @@ registry-imagestream-meta dt:after {
 
 .image-sources {
   .source-path:after,
-  .destination-dir:before {
+  .secrets-location:before {
     content: "\00a0 ";
```

Comment:
I'm a bit confused by this style. `\00a0` appears to be a non-breaking space, but then we're adding a regular space right after? Doesn't that defeat the purpose of the non-breaking space?